### PR TITLE
Community meeting notes landing

### PR DESCRIPTION
**Pull request by cgiradkar**

### CODE REVIEW

**Review by Luap99 — COMMENTED** [review]
Note all our projects require a DCO signoff in the commit message, first make sure to configure the git user and email.
```
git config --global user.name "<your real name>"
git config --global user.email "<your email>"
```

Then when you make commits remember to add the `-s` option, i.e. `git commit -s`

To add the signoff to your existing commits use `git rebase --signoff HEAD~5` where 5 is the number of your commits in your working branch.

**Comment by mairin** [comment]
@cgiradkar this is looking great! One nit - the list of archived meetings has transparent lines through it:

Screenshot from 2023-06-14 06-57-31
Screenshot from 2023-06-14 06-57-25

File: static/data/community.ts [hunk 1]
```diff
@@ -91,15 +91,15 @@ const communityMeetings = {
...
+          date: 1635814871000,
           icon: 'film',
           buttons: [
             { text: 'Watch recordings', path: MEETING_URL },
             { text: 'Meeting Minutes', path: 'https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both' },
           ],
         },
         {
-          date: '1633395671',
+          date: 1633395671000,
```
Comment (mairin) [inline static/data/community.ts:102]: Would it be possible to use a more human-friendly date format and have it be converted when it's read in / consumed from here?

**Comment by cgiradkar** [comment]
Closing PR as the PR got polluted in doing signed interactive rebase

**Comment by cgiradkar** [comment]
new PR here: https://github.com/containers/podman.io/pull/163

**Pull request closed without merging**